play/apps/play/models.py

```diff
@@ -0,0 +1,41 @@
+from apps.play.fields import ShortUUIDField
+from django.db import models
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.base_user import AbstractBaseUser
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+    id = ShortUUIDField(prefix="user", max_length=128, primary_key=True)
+    team_id = models.CharField(max_length=128)
+
+    # is_admin = models.BooleanField(default=False)
+    is_bounty = models.BooleanField(default=False)
+    is_registration = models.BooleanField(default=False)
+    is_tournament = models.BooleanField(default=False)
+
+    USERNAME_FIELD = 'id'
+    REQUIRED_FIELDS = []
+
+    class Meta:
+        app_label = 'play'
+
+
+class Team(models.Model):
+    id = ShortUUIDField(prefix="team", max_length=128, primary_key=True)
+    name = models.CharField(max_length=128)
+    description = models.TextField()
+    tournament_snake_id = models.CharField(max_length=128)
```

Comment:
I think it's too early to be putting tournament stuff in, let's get the site working first and running games, before putting in anything related to running tournaments.  Especially since I think the model makes sense to go the other direction, i.e. the tournament has links to the snake.

Reply:
his wasn't really tournament logic, more of a "one snake per team" type logic. Makes sense to not introduce yet, and would be gameplay logic derived from a game master or something. I will happily remove.